interface/new/new_comprehensive.php

```diff
@@ -98,6 +99,23 @@ function getSearchClass($data_type) {
 <?php include_once("{$GLOBALS['srcdir']}/options.js.php"); ?>
 <link rel="stylesheet" type="text/css" href="../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
 
+<?php
+    //this will hold the javascript submit me param new_validate;
+    $new_validate = $GLOBALS['new_validate'] ? 1 : 0;
+    if($GLOBALS['full_new_patient_form'] == '4')//use hook of patient validation = 4
+    {
+    $hook = checkIfPatientValidationHookIsActive();
+    }
+    else {
+    print "<script>";
+        print "$(document).ready(function(){
+        $(\"#DEM\").submit(function(e){
+        submitme(" . $new_validate . ",e,\"DEM\"); 
+        })});";
+        print "</script>";
+    }
```

Comment:
I removed the above else block and everything still works. Is there a reason that this is here that I am missing?